lxd/storage_buckets.go

```diff
@@ -602,6 +592,7 @@ func storagePoolBucketPut(d *Daemon, r *http.Request) response.Response {
 			return response.SmartError(err)
 		}
 
+		// FIXME: ETag check?
```

Comment:
There doesn't appear to be an eTag check for updating a storage bucket.

Reply:
I've added an issue to track this here: https://github.com/canonical/lxd/issues/13549